### DISCUSSION TITLE
Please remove the detonator sound. The sound is really annoying. Add output pin in the detonator that broadcasts 1 after something exploded.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15066] [Ideas and suggestions]
Detonators are used for foam grenades if there is pressure in the compartment.
And if the hull is breached again, and the grenade is already empty, then the detonator beeps continuously while there is pressure in.
Another option is to let detonator beep once, only if something exploded in it.
Or lets make a setting "turn off the detonator sound" in the detonator itself or in the game options.
Another improvement that would be great is:
Make an output signal for the detonator that broadcasts 1 after something exploded in the detonator, and resets to 0 if the contents of the detonator are replaced or removed.
Then it will be easy to make an indicator "grenade replacement required" from one lighting component and one wire.
Now this indicator requires a signal check component and three additional wires and a button to reset the state.

As alternative you can make a separate type of detonator with a built-in light indicator that lights up red after triggering and turns off 
if the contents are replaced or via a checkbox.

**Comment by NoahRoopull** [comment]
This is a good idea in theory, but I think it should probably just play the sound once instead. Being able to silence the timed detonator opens up far too many options for griefing.
